2-repositories/iglu-server/src/main/scala/com/snowplowanalytics/iglu/server/storage/Storage.scala

```diff
@@ -80,11 +80,21 @@ object Storage {
     config match {
       case StorageConfig.Dummy =>
         Resource.liftF(storage.InMemory.empty)
-      case StorageConfig.Postgres(host, port, name, username, password, driver, threads) =>
-        val url = s"jdbc:postgresql://$host:$port/$name"
+      case StorageConfig.Postgres(host, port, name, username, password, driver, threads, maxPoolSize) =>
         for {
           connectEC <- ExecutionContexts.fixedThreadPool(threads.getOrElse(32))
-          xa <- HikariTransactor.newHikariTransactor[F](driver, url, username, password, connectEC, transactEC)
+          _ <- Resource.liftF(Async[F].delay(Class.forName(driver)))
+          xa <- HikariTransactor.initial[F](connectEC, transactEC)
+          _ <- Resource.liftF {
+            xa.configure { ds =>
```

Comment:
Why don't we use existing `newHikariTransactor` function and then not just adding `setMaximumPoolSize`?